<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendFileWindow</class>
 <widget class="QMainWindow" name="SendFileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send Files</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="label_current_user">
      <property name="text">
       <string>Logged in as: USERNAME</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="group_send">
      <property name="title">
       <string>Send File</string>
      </property>
      <layout class="QGridLayout" name="layout_send">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="1" column="1">
        <widget class="QComboBox" name="combo_receiver"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_receiver">
         <property name="text">
          <string>To user:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="spin_minutes">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1440</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLabel" name="lbl_file">
         <property name="text">
          <string>No file selected</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="btn_send">
         <property name="styleSheet">
          <string notr="true">
QPushButton {
    background-color: #0077ff;
    color: white;
    border-radius: 8px;
    padding: 6px;
}
</string>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_minutes">
         <property name="text">
          <string>Expiration (minutes):</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="btn_select">
         <property name="styleSheet">
          <string notr="true">
background-color: #0077ff;
color: white;
border-radius: 8px;
padding: 6px;
</string>
         </property>
         <property name="text">
          <string>Select File</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="group_incoming">
      <property name="title">
       <string>Files Sent to You</string>
      </property>
      <layout class="QGridLayout" name="layout_incoming">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_refresh">
         <property name="styleSheet">
          <string notr="true">
background-color: #e6f2ff;
color: #0077ff;
border-radius: 8px;
padding: 6px;
</string>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="input_filter">
         <property name="placeholderText">
          <string>Filter...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QListWidget" name="list_incoming"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_download">
         <property name="styleSheet">
          <string notr="true">
background-color: #0077ff;
color: white;
border-radius: 8px;
padding: 6px;
</string>
         </property>
         <property name="text">
          <string>Download</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
